andela/apollo-ah-backend
# Feat/164198180/signup email

**What does this PR do?**
This PR enables a user to get a mail with a link that he can use to confirm his account

**Tasks to be completed?**
- configure nodemailer
- generate confirmation link
- configure moching for emails
- write tests 
- handle account confimation
- handle errors and success messages

**How can this be manually tested?**
- clone the repo
- install node js, yarn  and postgress 
- run `yarn install` to install dependencies
- create .env from .env.example
- create .env.test from .env.example and configure
- create a database on Postgres and add the db name in .env
- run `yarn run migrate`
- run `yarn run start: dev`
- send a POST request to `/api/v1/users/` to register a user, the user should get an email, click the link in the mail and the user gets confirmed

**Relevant PT stories?**
`164198180`
